libopenage/gui/guisys/public/gui_renderer.h

```diff
@@ -4,11 +4,7 @@
 
 #include <memory>
 
-#ifndef __APPLE__
-#include <GL/gl.h>
-#else
-#include <OpenGL/gl.h>
-#endif
+#include <QtGui/qopengl.h>
```

Comment:
[A Qt examples page](http://doc.qt.io/qt-5/qtopengl-hellogl2-logo-h.html) shows that it can be used directly [Sorry, they don't have line numbers; and I don't know the protocol for sharing their code without involving the copyrights/licences]. Given, I used a more explicit full path - which I can change to the shorter one if needed. OTOH, `QOpenGLFunctions` provides some wrappers which aren't required. I'm not well versed with Qt to take a decision either way.

I observed the pimpl idiom used in the gui code (here and in a few other places). For the inclusion of decided-upon file in `gui_renderer.cpp`:
Option 0: Before `gui_renderer.h` directly contradicts [mom's recommendation](https://github.com/SFTtech/openage/blob/d94876028d28f80a1ee7e72846133e6d7fdc9965/doc/code_style/mom.cpp#L17-L18). I'd prefer not to do that.
Option 1: After `gui_renderer.h` results in compilation failure for all units including it (`error C3646: 'render': unknown override specifier` and some others). :x:

Option (try larger change): Change/remove the `GuiRenderer::render` member to make it work with a forward-declared type `GLuint`. I'll need more information to make this work.
Option (none of these): Revert back to the expanded form before my last change.